web-services/model/src/main/java/datawave/webservice/query/model/ModelKeyParser.java

```diff
@@ -105,16 +102,14 @@ public static Mutation createDeleteMutation(FieldMapping mapping, String modelNa
         ColumnVisibility cv = new ColumnVisibility(mapping.getColumnVisibility());
         Mutation m;
         String dataType = StringUtils.isEmpty(mapping.getDatatype()) ? "" : NULL_BYTE + mapping.getDatatype().trim();
-        String indexOnly = mapping.getIndexOnly() ? NULL_BYTE + INDEX_ONLY : "";
+        
         if (Direction.REVERSE.equals(mapping.getDirection())) {
             m = new Mutation(mapping.getFieldName());
-            m.putDelete(modelName + dataType, mapping.getModelFieldName() + indexOnly + NULL_BYTE + mapping.getDirection().getValue(), cv,
-                            System.currentTimeMillis());
+            m.putDelete(modelName + dataType, mapping.getModelFieldName() + NULL_BYTE + mapping.getDirection().getValue(), cv, System.currentTimeMillis());
```

Comment:
We may need to add two delete mutations here: 1 for the original model entry and 1 for the new model entry.

Reply:
We only need the two delete mutations (with and without index_only) in the forward section (the else block). The change in this section is fine.

Reply:
> 
> 
> We only need the two delete mutations (with and without index_only) in the forward section (the else block). The change in this section is fine.

It doesn't appear this has been addressed yet.